router-tests/testenv/testenv.go

```diff
@@ -1202,6 +1202,13 @@ func (e *Environment) GraphQLServeSentEventsURL() string {
 	return u.String()
 }
 
+func (e *Environment) GraphQLMultipartResponsesURL() string {
+	u, err := url.Parse(e.GraphQLRequestURL())
+	require.NoError(e.t, err)
+	u.RawQuery = core.WgMultipartParam
```

Comment:
This is not the way how protocol negotiation is done. The user sends the `Accept: multipart/mixed;subscriptionSpec="1.0", application/json` header to use subscriptions over multipart.